src/main.rs

```diff
@@ -153,6 +203,19 @@ fn work() -> Result<(), Cue2CCDError> {
         return Err(Cue2CCDError::MissingFilesError { missing_files });
     }
 
+    let mut sbi_lba_array: Vec<i32> = Vec::new();
+    let mut sbi_data: Vec<Vec<u8>> = Vec::new();
+
+    // TODO: is this extension check case sensitive?
+    if Path::new(&output_stem.with_extension("sbi")).exists() {
```

Comment:
Could probably assign `Path::new(&output_stem.with_extension("sbi"))` to a variable since it's used twice?

Reply:
True, I considered that. I'll go add that now.

Reply:
Discussed in discord, there isn't really a more clean way to do this, so leaving unchanged.